sdk/core/azure-core/src/http/curl/curl_connection_pool.hpp

```diff
@@ -9,9 +9,10 @@
 
 #pragma once
 
-#include "azure/core/http/curl/curl_connection.hpp"
 #include "azure/core/http/http.hpp"
 
+#include "curl_connection.hpp"
```

Comment:
Add _private suffix to be more explicit about intent.
```suggestion
#include "curl_connection_private.hpp"
```

Do the same for the other private headers in the `/src/` directory.

Reply:
Let me take care of that on a new PR to include all other headers over there in Core